src/app/config/config.c

```diff
@@ -6523,10 +6533,11 @@ port_parse_config(smartlist_t *out,
   }
 
   retval = 0;
- err:
+err:
```

Comment:
Our coding style has labels indented by 1 space.

So it's best to just leave this line as it was.

Reply:
Hi, please put one space before labels.
```suggestion
 err:
```

Reply:
(Actually, I fixed the label in my branch, so don't worry about it.)